app/code/Magento/Backend/Api/Search/ItemsInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Backend\Api\Search;
+
+use Magento\Backend\Model\Search\SearchCriteria;
+
+/**
+ * @api
+ */
+interface ItemsInterface
```

Comment:
I suggest renaming this interface, as Items is too broad definition. 
Also `getResults` method in `ItemsInterface` creates confusion, since there may not be results of items.